test/alerts/alerts_test.exs

```diff
@@ -7,8 +7,6 @@ defmodule AlertsTest do
   alias Alerts.Alert
   alias Alerts.InformedEntity, as: IE
 
-  @now Util.to_local_time(~N[2018-01-15T12:00:00])
```

Comment:
📓 There was a warning when running tests that this was no longer being used so I removed it.